README.md

```diff
@@ -115,6 +115,19 @@ Where a theme requires specific icons that are not part of the Font Awesome set,
 `px vs. em` - we specify font sizes in `px`.
 When creating a widget, spacing (margin, padding) should be specified using `px` unless the design calls for proportional spacing, in which case `em` can be used.
 
+### Z-index layering
+
+Widgets adhere to a basic convention for blocking off ranges of z-index values based on function and visual context. This is present in both individual widget CSS and in the Dojo theme styles. These values can be overridden in a custom theme if necessary, since no `z-index` values are set in fixed styles.
```

Comment:
"blocking off" -> "using specific"
"This is present" -> "This convention is followed"